README.md

```diff
@@ -1,52 +1,77 @@
 
 # Microsoft Azure Cloud Shell Image
 
-Azure Cloud Shell is a browser-based shell environment which enables Azure customers to manage and configure their Azure services. It provides a [host of tools](https://docs.microsoft.com/azure/cloud-shell/features), including Azure CLI, Azure PowerShell, Ansible, Terraform, Puppet Bolt, kubectl, and many more.
-
-For more details, check out [Overview of Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/overview#:~:text=Features%201%20Browser-based%20shell%20experience.%20...%202%20Choice,7%20Connect%20your%20Microsoft%20Azure%20Files%20storage.%20).
-
-
-# About this repository
-
-When you connect to Azure Cloud Shell, we start a container containing a wide variety of tools, and connect your
-browser to a shell process running inside that container. This repository contains the Docker files used to build that image. 
-It does _not_ contain all of the code used for the rest of the Azure Cloud Shell service. The code in this repository may not 
-match exactly to what is running in the Cloud Shell service at any given time. The service is updated periodically and changes 
-are gradually rolled out to different regions over time, so there may be a lag of up to 3-4 weeks between a change being made 
-here and being reflected in all Cloud Shell regions.
+Azure Cloud Shell is an interactive, authenticated, browser-accessible shell for managing Azure
+resources. As a web-based environment, Cloud Shell provides immediate management access to any
+device with a network connection. Azure Cloud Shell provides a

```

Comment:
"any device"? Should that be "any Azure connected device"?

Reply:
Yes it should :)